.github/workflows/profiling.yaml

```
@@ -0,0 +1,101 @@
+name: Profiling Comparison for Specific File 2
+
+on:
+  push
+jobs:
+  profiler:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checkout the code from the repository with full history
+    - name: Checkout code
+      uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)